rules-tests/Php70/Rector/If_/IfToSpaceshipRector/Fixture/skip.php.inc

```diff
@@ -30,14 +30,6 @@ class Skip
             return ($a[0] <= $b[0]) ? 1 : -1;
         });
 
-        usort($languages, function ($a, $b) {
-            if ($a[0] === $b[0]) {
-                return 0;
-            }
-
-            return ($a[0] < $b[0]) ? -1 : 1;
-        });
```

Comment:
I removed above skip as it seems applied:

```diff
-            if ($a[0] === $b[0]) {
-                return 0;
-            }
-
-            return ($a[0] < $b[0]) ? -1 : 1;
+            return $a[0] <=> $b[0];
         });
```

added to new fixture https://github.com/rectorphp/rector-src/pull/1676/commits/d0e7b445507849607c50b42206a87ad4cf1cb18e